### PR TITLE
Allow to cancel premium DO requests

### PR DESCRIPTION
**Pull request by Jesus89**

### CODE REVIEW

File: lib/assets/javascripts/new-dashboard/components/Catalog/DatasetHeader.vue [hunk 1]
```diff
@@ -166,11 +173,18 @@ export default {
     getFormURL () {
       return formURL(this.dataset);
     },
-    interesed () {
+    async interesed () {
```
Comment (VictorVelarde) [inline lib/assets/javascripts/new-dashboard/components/Catalog/DatasetHeader.vue:176]: typo interested?

Comment (Jesus89) [reply lib/assets/javascripts/new-dashboard/components/Catalog/DatasetHeader.vue:176]: True

Comment (Jesus89) [reply lib/assets/javascripts/new-dashboard/components/Catalog/DatasetHeader.vue]: Fixed

**Review by VictorVelarde — APPROVED** [review]
LGTM!

**Pull request merged**